app/models/name/taxonomy.rb

```diff
@@ -35,6 +35,12 @@ def self.ranks_below_species
     [:Form, :Variety, :Subspecies]
   end
 
+  # Does this Name have a rank above genus that's in MO's Classification scheme
+  # Currently all ranks above :Genus except :Group
+  def classified_above_genus?
+    [:Family, :Order, :Class, :Phylum, :Kingdom, :Domain].freeze.include?(rank)
```

Comment:
How is this different from !at_or_below_genus? ?  Seems like we should remove one or the other.

Reply:
Good point. I didn't notice that. I'll change it.